externalpolicy/builtin_properties.go

```diff
@@ -55,7 +55,19 @@ func CreateNodeBuiltInPolicy(availableMem bool, omitGenHwId bool, existingPolicy
 	if existingPolicy != nil && existingPolicy.Properties.HasProperty(PROP_NODE_PRIVILEGED) {
 		privProp, _ := existingPolicy.Properties.GetProperty(PROP_NODE_PRIVILEGED)
 		glog.Infof("Found existing policy with %v", privProp)
-		privileged = privProp.Value.(bool)
+		var ok bool
```

Comment:
Do we need the Info msg on line 57? maybe it's ok to make it a .V(5) msg.
I also noticed that the errors in here are logged as info, they should be glog.Errorf(...) like you have on line 68

Reply:
I changed it to a .V(5) . The errors here need to be .V(x) since this fuction can be called through the cli with hzn dev service start.